docs/content/documentation/getting-started/cli-usage.md

```diff
@@ -46,7 +46,7 @@ $ zola build --base-url $DEPLOY_URL
 This is useful for example when you want to deploy previews of a site to a dynamic URL, such as Netlify
 deploy previews.
 
-You can override the default output directory `public` by passing another value to the `output-dir` flag.
+You can override the default output directory `public` by passing another value to the `output-dir` flag (if this directory already exists, it is overwritten).
```

Comment:
deleted is a better description than overwritten

Reply:
Done! :)